src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/02 Advanced Examples.md

```diff
@@ -3,11 +3,19 @@ title: 'Advanced Examples'
 excerpt: 'Advanced Examples using the k6 Scenario API - Using multiple scenarios, different environment variables and tags per scenario.'
 ---
 
-## Using multiple scenarios
+You can use multiple scenarios in one script, and these scenarios can be run in sequence or in parallel.
+Some ways that you can combine scenarios include the following:
+- Have different start times to sequence workloads
+- Add per-scenario tags and environment variables
+- Make scenario-specific thresholds.
+- Export multiple scenarios as functions in [VU code](/using-k6/test-lifecycle).
```

Comment:
```suggestion
- Execute different exported functions in different scenarios, [not just the `default` function](https://k6.io/docs/using-k6/test-lifecycle/#the-vu-stage).
```

And https://k6.io/docs/using-k6/test-lifecycle/#the-vu-stage itself should probably be slightly expanded in another PR, to contain a mention (with a link here) about the same capability of scenarios.

And, this should probably be a separate issue and a bigger discussion, but "VU code" and "VU stage" in https://k6.io/docs/using-k6/test-lifecycle/ is probably more confusing than helpful :thinking: It is conflating the terms, "VU" has one meaning in every other context and another meaning in the "test lifecycle" document :thinking: 

It's probably more accurate to say "test/VU initialization context" and "test execution context" :thinking: Again, though, this is a bigger discussion out of scope for this PR.

Reply:
I worry this suggestion is a little to extract. IMO, the "killer feature" here is that different scenarios can execute different test logic. Do you agree? If so, how about

```diff
- Export multiple scenarios as functions in [VU code](/using-k6/test-lifecycle).
+ Use multiple scenarios to run different test logic, so that VUs don't run in only the `default` function](https://k6.io/docs/using-k6/test-lifecycle/#the-vu-stage).
```

Reply:
:+1: yeah, this is better, though you can drop the `in` in of `run in only the default`, it should be `run only the default`